internal/app/app.go

```diff
@@ -0,0 +1,41 @@
+package app
+
+import (
+	"github.com/ActiveState/cli/internal/constants"
+	configMediator "github.com/ActiveState/cli/internal/mediators/config"
+)
+
+func init() {
+	configMediator.RegisterOption(constants.AutostartSvcConfigKey, configMediator.Bool, configMediator.EmptyEvent, configMediator.EmptyEvent)
```

Comment:
This should be in the autostart package.

Reply:
Turns out it already is and this is unnecessary.